Lib/multiprocessing/util.py

```diff
@@ -410,6 +410,13 @@ def _flush_std_streams():
 #
 
 def spawnv_passfds(path, args, passfds):
+    if 'vxworks' in sys.platform:
+        import subprocess
+        try:
+            return subprocess.Popen(args).pid
+        except:
```

Comment:
This is not good.  First, 'try' is bare 'except' is bad news, hides bugs.  Second, you are doing the 'vxworks' test on every function call, for every platform.  Instead you could have something like:
```
def spawnv_passfds(...):
    ...

if 'vxworks' in sys.platform:
  # a comment explaining why this is needed
  def spawnv_passfds(...):
    # implementation of vxworks special version
```